stylesheets/_component.navigation.scss

```diff
@@ -219,6 +219,7 @@ $nav-color-dark:  darken(color(jadu-blue, dark), 5%) !default;
 .nav-primary {
     background-color: $nav-color-light;
     box-shadow: inset 0 0 0 $nav-color-dark;
+    overflow-y: hidden;
```

Comment:
Will this prevent the primary nav from scrolling vertically if the number of items is greater than the viewport height?